client/components/fields/editor/AssociatedEventWrapper.tsx

```diff
@@ -22,7 +27,7 @@ export class AssociatedEventField extends React.PureComponent<IAssociatedEventFi
     }
 }
 
-const mapStateToProps = (state) => ({
+const mapStateToProps = (state): IReduxStoreProps => ({
```

Comment:
locked items and files appear unused to me. It's not used inside `EditorFieldAssociatedEventComponent` and I don't see that entire props object would be passed to children.

Reply:
seems you still getting files from store when they're not being used?